gresources/nemo-file-management-properties.glade

```diff
@@ -1234,6 +1234,23 @@ along with .  If not, see <http://www.gnu.org/licenses/>.
                                         <property name="can-focus">False</property>
                                         <property name="orientation">vertical</property>
                                         <property name="spacing">6</property>
+                                        <child>
+                                          <object class="GtkRadioButton" id="scripts_execute_terminal_radiobutton">
+                                            <property name="label" translatable="yes">_Run executable text files in the terminal when they are opened</property>
```

Comment:
You can get rid of the _ in front of Run - this would normally indicate that pressing R should select this option. In reality these shortcuts don't work on this page, so we can leave it out (don't change the other options, as they've already been translated, and any change will void those translations).